case-studies/bayren.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: case-study
+
+client: bayren
+
+title: Bay Area Regional Energy Network (BayREN)
+
+tags: [Our Work]
+
+description: "An example of our work – an eTool - for the Bay Area Regional Energy Network."
+
+client-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
+
+challenge-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
+
+technologies: [javascript, angular, node, docker, kubernetes, mysql]
+
+outcome-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
```

Comment:
Also this. Should this be different from the client-text?

Reply:
Fixed.